doc[iframe]: remove `HTMLDialogElement.showModal` from `allow-popups` (#39580)
mdn/content: files/en-us/web/html/reference/elements/iframe/index.md

    - `allow-pointer-lock`
      - : Allows the page to use the [Pointer Lock API](/en-US/docs/Web/API/Pointer_Lock_API).
    - `allow-popups`

>       - : Allows popups (like from {{domxref("Window.open()")}}, `target="_blank"`). If this keyword is not used, that functionality will silently fail.

Reviewer: ```suggestion
      - : Allows popups (created, for example, by {{domxref("Window.open()")}} or `target="_blank"`). If this keyword is not used, such functionality will silently fail.
```
